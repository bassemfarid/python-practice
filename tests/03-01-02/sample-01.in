John
Henry
Doe
William
A girl has no name