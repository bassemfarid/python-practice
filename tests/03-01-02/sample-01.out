hello John!
NEXT!
hello Henry!
NEXT!
hello Doe!
NEXT!
hello William!
NEXT!
*sweats profusely*